spring-cloud-kubernetes-discovery/src/main/java/org/springframework/cloud/kubernetes/discovery/KubernetesServiceInstance.java

```diff
@@ -31,6 +31,11 @@
  */
 public class KubernetesServiceInstance implements ServiceInstance {
 
+	/**
+	 * Key of the namespace metadata.
+	 */
+	public static final String NAMESPACE_METADAT_KEY = "k8s_namespace";
```

Comment:
I dont think we need to prefix the key with `k8s` since this is Kubernetes specific

Reply:
Ah right, the whole metadata part is Kubernetes specific. I removed the prefix now.